tradeit-android-sdk/src/androidTest/kotlin/it/trade/android/sdk/manager/TradeItLinkedBrokerManagerTest.kt

```diff
@@ -0,0 +1,97 @@
+package it.trade.android.sdk.manager
+
+import android.content.Context
+import android.content.SharedPreferences
+import android.support.test.InstrumentationRegistry
+import android.support.test.runner.AndroidJUnit4
+import android.test.suitebuilder.annotation.LargeTest
+import android.util.Log
+
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.TimeUnit
+
+import it.trade.android.sdk.TradeItConfigurationBuilder
+import it.trade.android.sdk.TradeItSDK
+import it.trade.android.sdk.internal.TradeItKeystoreService
+import it.trade.model.TradeItErrorResult
+import it.trade.model.callback.TradeItCallback
+import it.trade.model.reponse.TradeItAvailableBrokersResponse
+import it.trade.model.request.TradeItEnvironment
+
+import android.support.test.InstrumentationRegistry.getInstrumentation
+import org.hamcrest.Matchers.`is`
+import org.hamcrest.Matchers.notNullValue
+import org.hamcrest.Matchers.nullValue
+import org.junit.Assert.assertThat
+
+@RunWith(AndroidJUnit4::class)
+@LargeTest
+class TradeItLinkedBrokerManagerTest {
```

Comment:
running androidTests on emulator with api level 19 got below error: 
```
Test running failed: Instrumentation run failed due to 'java.lang.ClassNotFoundException'
```
do you have any idea what went wrong? tests was ok in api level 22 and 28 emulators though.

Reply:
Not sure. I tested on develop and the error was already there